<?xml version="1.0"?>
<!--
  Launch wit driver
-->
<launch>
  


<!-- run node -->
  <node name="magnetic_field_vector1" pkg="magnetic_field_localization" type="magnetic_field_vector" output="screen">
    <param name="magnetometer_topic" type="str" value="/imu_magnetic0" />
    <param name="vector_topic" type="str" value="/magnetic_vector0" />
    <param name="magnetic_field_frequency" type="int" value="50" />
    <param name="number_of_points_for_analysis" type="int" value="50" />
    <param name="number_of_cycles_for_analysis" type="int" value="1" />
  </node>

  <node name="magnetic_field_vector2" pkg="magnetic_field_localization" type="magnetic_field_vector" output="screen">
    <param name="magnetometer_topic" type="str" value="/imu_magnetic1" />
    <param name="vector_topic" type="str" value="/magnetic_vector1" />
    <param name="magnetic_field_frequency" type="int" value="50" />
    <param name="number_of_points_for_analysis" type="int" value="50" />
    <param name="number_of_cycles_for_analysis" type="int" value="1" />
  </node>

</launch>
